filament/backend/test/test_LoadImage.cpp

```diff
@@ -546,8 +581,18 @@ TEST_F(LoadImageTest, UpdateImage3D) {
 
         api.bindDescriptorSet(descriptorSet, 0, {});
 
-        renderTriangle({ { shader.getDescriptorSetLayout() } },
-                defaultRenderTarget, swapChain, shader.getProgram());
+        RenderPassParams params = getClearColorRenderPass();
+        params.viewport.width = 512;
```

Comment:
nit: I'd add a constant for 512.

Reply:
The tests are absolutely littered with 512s, I've actually got another note in my notebook to clean them all up.
So I'll take care of that next.